grammars/Babel Language.json

```diff
@@ -935,7 +935,7 @@
         },
         {
           "comment": "GraphQL ( Relay.QL ) supprt. Use two forms of scopes! fixes some themes",
-          "begin": "\\s*+(?:((Relay)(.)(QL))|(gql|graphql)|(/\\* GraphQL \\*/))\\s*((`))",
+          "begin": "\\s*+(?:((Relay)(.)(QL))|(gql|graphql|graphql.experimental)|(/\\* GraphQL \\*/))\\s*((`))",
```

Comment:
If you add a back slash can you also add one to the Relay.QL regex as that's an error on my part.